finality-aleph/src/party/authority.rs

```diff
@@ -78,11 +80,12 @@ impl Subtasks {
             _ = &mut self.exit => false,
             _ = self.member.stopped() => true,
             _ = self.aggregator.stopped() => true,
-            _ = self.forwarder.stopped() => true,
             _ = self.refresher.stopped() => true,
             _ = self.data_store.stopped() => true,
         };
+        debug!(target: "aleph-party", "Something died and it was unexpected: {:?}", result);
```

Comment:
I would make this log conditional on `result`, usually its not that useful.

Reply:
Done